build/Helix/ConvertWttLogToXUnit.cs

```diff
@@ -0,0 +1,287 @@
+            assembly.SetAttributeValue("skipped", 0);
+            assembly.SetAttributeValue("time", (int)testPass.TestPassExecutionTime.TotalSeconds);
+            assembly.SetAttributeValue("errors", 0);
+            root.Add(assembly);
+
+            var collection = new XElement("collection");
+            collection.SetAttributeValue("total", resultCount);
+            collection.SetAttributeValue("passed", passedCount);
+            collection.SetAttributeValue("failed", failedCount);
+            collection.SetAttributeValue("skipped", 0);
+            collection.SetAttributeValue("name", "Test collection");
+            collection.SetAttributeValue("time", (int)testPass.TestPassExecutionTime.TotalSeconds);
+            assembly.Add(collection);
+
+            foreach (var result in results)
+            {
+                var test = new XElement("test");
+                test.SetAttributeValue("name", result.Name);
+
+                var className = result.Name.Substring(0, result.Name.LastIndexOf('.'));
+                var methodName = result.Name.Substring(result.Name.LastIndexOf('.') + 1);
+                test.SetAttributeValue("type", className);
+                test.SetAttributeValue("method", methodName);
+
+                test.SetAttributeValue("time", result.ExecutionTime.TotalSeconds);
+                test.SetAttributeValue("result", result.Passed ? "Pass" : "Fail");
+
+                if (!result.Passed)
+                {
+                    var failure = new XElement("failure");
+                    failure.SetAttributeValue("exception-type", "Exception");
+
+                    var message = new XElement("message");
+                    message.Add(new XCData(result.Details));
+                    failure.Add(message);
+
+                    test.Add(failure);
+                }
+
+                collection.Add(test);
+            }
+
+            File.WriteAllText(xunitOutputPath, root.ToString());
+        }
+        public static TestPass ParseTestWttFile(string fileName, bool cleanupFailuresAreRegressions)
+        {
+            using (var stream = System.IO.File.OpenRead(fileName))
+            {
+                var doc = XDocument.Load(stream);
+                var testResults = new List<TestResult>();
+                var testExecutionTimeMap = new Dictionary<string, List<double>>();
+
+                TestResult currentResult = null;
+                long frequency = 0;
+                long startTime = 0;
+                long stopTime = 0;
+                bool inTestCleanup = false;
+
+                bool shouldLogToTestDetails = false;
+
+                long testPassStartTime = 0;
+                long testPassStopTime = 0;
+
+                Func<XElement, bool> isScopeData = (elt) =>
+                {
+                    return
+                        elt.Element("Data") != null &&
+                        elt.Element("Data").Element("WexContext") != null &&
```

Comment:
Can it be elt.Element("Data")?.Element("WexContext") != null

Reply:
I'll make the change.

Reply:
Hmmm. Looks like we are using an older c# compiler here. That is hitting a compiler error: 
c:\Users\kmahone\AppData\Local\Temp\r05tw5ul.0.cs(115) : Invalid expression term '.'
c:\Users\kmahone\AppData\Local\Temp\r05tw5ul.0.cs(114) :                     return
c:\Users\kmahone\AppData\Local\Temp\r05tw5ul.0.cs(115) : >>>
elt.Element("Data")?.Element("WexContext") != null &&
c:\Users\kmahone\AppData\Local\Temp\r05tw5ul.0.cs(116) :                         (
At D:\microsoft-ui-xaml\build\Helix\ConvertWttLogToXUnit.ps1:9 char:1

I'll switch back.